relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -1232,4 +1238,51 @@ mod tests {
         // Can be scrubbed with db system.
         assert_eq!(scrubbed.0.as_deref(), Some("my-component-name"));
     }
+
+    #[test]
+    fn scrub_allowed_host() {
+        let examples = &[
+            (
+                "https://foo.bar.internal/api/v1/submit",
+                &["foo.bar.internal".to_string()],
+                "https://foo.bar.internal",
+            ),
+            (
+                "http://192.168.1.1:3000",
+                &["192.168.1.1".to_string()],
+                "http://192.168.1.1:3000",
+            ),
+            (
+                "http://[1fff:0:a88:85a3::ac1f]:8001/foo",
+                &["[1fff:0:a88:85a3::ac1f]".to_string()],
+                "http://[1fff:0:a88:85a3::ac1f]:8001",
+            ),
+        ];
```

Comment:
```suggestion
        let examples = [
            (
                "https://foo.bar.internal/api/v1/submit",
                ["foo.bar.internal".to_string()],
                "https://foo.bar.internal",
            ),
            (
                "http://192.168.1.1:3000",
                ["192.168.1.1".to_string()],
                "http://192.168.1.1:3000",
            ),
            (
                "http://[1fff:0:a88:85a3::ac1f]:8001/foo",
                ["[1fff:0:a88:85a3::ac1f]".to_string()],
                "http://[1fff:0:a88:85a3::ac1f]:8001",
            ),
        ];
```

This should help get rid of the clippy allow.